src/utils/generator.js

```diff
@@ -1068,13 +1126,21 @@ export class Generator {
 
   // PITFALL Warning: adding a new kind of TemporalBuildNodeEntry that is not the result of a join or composition
   // will break this purgeEntriesWithGeneratorDepencies.
```

Comment:
Let's talk about the PITFALL warning. You are adding a new kind of `TemporalBuildNodeEntry`, but there is no `purgeEntriesWithGeneratorDepencies`. Seems like I added that comment a while ago. I have no idea what this is supposed to mean now. Can you delete the PITFALL warning?